style.css

```diff
@@ -19,7 +20,65 @@ body {
   height: 60px;
 }
 
+.jsmenu {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 40px;
+  padding-right: 15px;
+}
+
+.ul-div {
+  display: none;
+  padding: 0;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  top: 50px;
+  transition: all 0.5s;
+  background: #3e3c3c;
+}
+
 .navilist {
+  display: flex;
+  flex-direction: column;
+}
+
+.navi-item {
+  text-align: left;
+}
+
+.navilist li a {
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff5e1;
+  text-decoration: none;
+  padding-top: 25px;
+  padding-bottom: 7px;
+  padding-left: 10px;
+  border-bottom: 1px solid #fff;
+}
+
+.ul-div ul {
+  flex-direction: column;
+  padding-right: 35px;
+}
+
+.ul-div.active {
+  display: flex;
+}
+
+.cancel-img {
+  display: none;
+  width: 50px;
+  height: 50px;
+  align-self: flex-end;
+  padding-right: 50px;
```

Comment:
- [optional] What did you think of reducing the padding-right to 40px, reducing a bit the image size?. Also, the horizontal line below each option is too pronounced. Please check once more the original design.